recipes/metaclock/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "metaclock" %}
+{% set version = "1.0.0" %}
+
+
+
+package:
+  name:  "{{ name }}"
+  version:  "{{ version }}"
+
+source:
+  url: https://github.com/SegataLab/metaclock/archive/{{ version }}.tar.gz
+  sha256: 146e6e8b5c8d62dc958a41cfb5a53cae9cc57bbc1da82d4020b217aafc180429
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+
+requirements:
+  host:
+    - python =3.6.10
+    - setuptools
+    - numpy
+    - cython
+    - pip
+  run:
+    - python =3.6.10
+    - numpy >=1.18.4
+    - matplotlib-base >=3.2.1
+    - seaborn >=0.10.1 
+    - samtools >=1.3.1
+    - bzip2 =1.0.8
+    - blast >=2.6.0
+    - biopython =1.76
+    - bowtie2 >=2.0.0
+    - pysam >=0.15.4
+    - mapdamage2 >=2.0.0
+    - trimal >=1.4.0
+    - cmseq >=1.0.2
+    - perl-bioperl
+    - prokka
+    - scikit-learn >=0.18.2
+    - ete3 >=3.1.1
+    - dendropy >=4.5.2
+    - raxml >=8.2.0
+test:
+  commands:
+    - metaclock_mac --help
+    - metaclock_visualizer --help
+    - metaclock_tailor --help
+    - metaclock_combiner --help
+
+about:
+  home: https://github.com/SegataLab/metaclock
+  license: A-GPL 3.0
+  license_file: LICENSE
+  summary: A python package for facilitating strain-level phylogenetic and molecular clock analysis
```

Comment:
Can you add a maintainers section (see some other recipes for an example) and list yourself there?